mapbox/mapbox-gl-js
# [docs] adds feedback component

This PR adds a feedback component to bottom of pages and sections. This component is currently on all docs.mapbox.com sites and it looks like this:

![image](https://user-images.githubusercontent.com/2180540/60911699-c6a0fa00-a251-11e9-9adf-3f6b68d5419b.png)


The component will appear at the end of:
- /mapbox-gl-js/overview/
- each subsection in /mapbox-gl-js/api/
- each example at /mapbox-gl-js/example/*/
- each section /mapbox-gl-js/style-spec/

Note: since the Style spec's subsections tend to be shorter than subsections on the API page, we decided to only place the component at the end of each Style spec section.

After the launch of this final site, look out for an internal post from the docs team where we'll share how teams can review ratings for given sites and pages. We'll also share how the docs team is triaging this feedback on a daily basis.

Ref https://github.com/mapbox/documentation/issues/209

## Launch Checklist
 - [x] check with gl-js team before merging since this PR makes changes to yarn.lock

### How to test

1. `yarn run start-docs`
1. click yes/no on one of the feedback components
1. open the Segment mapbox-staging debugger and search for "sent docs feedback" to see if your events came through (events should come through regardless of ad blocker)
